katas/square-digits/victor/squareDigits.js

```diff
@@ -1,3 +1,14 @@
 export function squareDigits(num) {
-    return 0;
-}
\ No newline at end of file
+        let  strNum = num.toString();
```

Comment:
please fix indentation (should bt 2 or 4 spaces in order to make sense with the nesting inside function block)